snapshots/artificial_intelligence/2025-11-07/energy_ai_iea.py

```diff
@@ -0,0 +1,38 @@
+"""Script to create a snapshot of dataset 'Energy and AI (IEA, 2025)'.
+
+INSTRUCTIONS FOR RUNNING THIS SNAPSHOT:
+
+Since the IEA dataset requires authentication, you need to manually download the file first:
+
+1. Visit: https://www.iea.org/data-and-statistics/data-product/energy-and-ai
+2. Create an IEA account or log in if you already have one
+3. Download the Excel file
+4. Place the downloaded file at: snapshots/artificial_intelligence/2025-11-07/energy_ai_iea.xlsx
```

Comment:
I think that you can place the file anywhere, as long as you then pass the same path to --path-to-file.